src/renderer/state.ts

```diff
@@ -808,7 +834,8 @@ export class AppState {
       | null
       | boolean,
   ) {
-    if (value) {
+    // null or undefined
+    if (value != null) {
```

Comment:
This is the one unrelated change I added. Previously, non-null falsy values would never get saved to localStorage.